bash/functions.sh

```diff
@@ -188,18 +183,11 @@ webimages() {
   echo 'Done.'
 }
 
-download_video() {
+# Download a YouTube video to the Desktop using `yt-dlp`
```

Comment:
Function naming inconsistency: `download_video` was renamed to `download-video` but other functions use underscores. Consider standardizing on one convention throughout the codebase.

Reply:
Will address this in a subsequent PR.

Reply:
<img src="https://media3.giphy.com/media/v1.Y2lkPWJkM2VhNTdlZGp1OTA2Z3UycmIzc2VocWVibnV2cndkbXNyMjRzcWdoNHFxMDV0MiZlcD12MV9naWZzX3RyZW5kaW5nJmN0PWc/xdH0MjQ83lGFVv7gjR/giphy.gif"/>